test/appsignal/probes/erlang_probe_test.exs

```diff
@@ -0,0 +1,33 @@
+defmodule Appsignal.Probes.ErlangProbeTest do
+  use ExUnit.Case
+
+  alias Appsignal.{FakeAppsignal, Probes.ErlangProbe}
+
+  setup do
+    {:ok, fake_appsignal} = FakeAppsignal.start_link()
+
+    [fake_appsignal: fake_appsignal]
+  end
+
+  describe "call/0" do
+    setup do
+      ErlangProbe.call()
+    end
+
+    test "gathers IO metrics", %{fake_appsignal: fake_appsignal} do
+      assert FakeAppsignal.get(fake_appsignal, "erlang_io")
```

Comment:
I think we need to test if the tags are set as well

Reply:
I believe @jeffkreeftmeijer and I decided to just add a smoke test here. (At first we were not planning on adding tests at all). I can take a look and see if I can do something with the tags.

Reply:
@jeffkreeftmeijer How would you build the key in that case?

Reply:
I'm not sure I understand your question. The key would remain the same, but you'd change `Appsignal.FakeAppsignal.set_gauge/3` to something like this:

``` elixir
 def set_gauge(key, value, tags) do
  update(__MODULE__, key, {value, tags})
end
```

Reply:
Yeah sure, but for some I send in 2 or more gauges with the same key but different tags. In this example they would get overridden. So no way to test if all of them are being sent.

Reply:
Added tests specifically to see if certain tags are here.